lib/src/web_driver.dart

```diff
@@ -217,21 +217,40 @@ class WebDriver implements SearchContext {
     }
   }
 
-  Future postRequest(String command, [params]) => _performRequest(
+  Future postRequest(String command, [params]) => _performRequestWithLog(
       () => _commandProcessor.post(_resolve(command), params),
       'POST',
       command,
       params);
 
-  Future getRequest(String command) => _performRequest(
+  Future getRequest(String command) => _performRequestWithLog(
       () => _commandProcessor.get(_resolve(command)), 'GET', command, null);
 
-  Future deleteRequest(String command) => _performRequest(
+  Future deleteRequest(String command) => _performRequestWithLog(
       () => _commandProcessor.delete(_resolve(command)),
       'DELETE',
       command,
       null);
 
+  Future _performRequestWithLog(
```

Comment:
Could you add a doc string to document how this is different from _performRequest() and when to use which?